Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionResultPositiveOtherWarningFragment.kt

```diff
@@ -6,6 +6,7 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.view.accessibility.AccessibilityEvent
+import androidx.appcompat.app.AlertDialog
```

Comment:
this import seems to be unused

Reply:
Hi Luka,

I am still making few changes in JUnit test. Please do not merge the code just yet.

Reply:
It's been taken care of